pkg/apis/config/sigstore_keys.go

```diff
@@ -0,0 +1,184 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/sigstore/policy-controller/pkg/apis/policy/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// SigstoreKeysConfigName is the name of ConfigMap created by the
+	// reconciler and consumed by the admission webhook for determining
+	// which Keys/Certificates are trusted for things like Fulcio/Rekor, etc.
+	SigstoreKeysConfigName = "config-sigstore-keys"
+)
+
+// Note that these are 1:1 mapped to public API SigstoreKeys. Reasoning
```

Comment:
Suggestion: let's add a link to the protos used of inspiration to create these API types.